### PR TITLE
Inline docs for threepio.py

### LINKED ISSUE #69: Write inline documentation for pythreepio/pythreepio/threepio.py
## Where?
[pythreepio/pythreepio/threepio.py](https://github.com/OpenMined/threepio/blob/master/pythreepio/pythreepio/threepio.py)

## Who?

Any developer who wants to work on Threepio should be able to read inline documentation for this file, as well as any file in the project.

## What?
We need to go "line-by-line" and create inline documentation for this file. Please try to use proper punctuation and English grammar to the best of your ability.

## Additional Context
None

### PR DESCRIPTION
**Pull request by Praful932**
## Description
- Added inline documentation and docstrings for threepio.py
- Fixed flake8 warnings
- removed deprecated function parameters

Resolves #69 

> I have followed restructured text format to document the code same as Pysyft docs

## Affected Dependencies
None

## How has this been tested?
Passed pytest and flake8 as described in the workflow file.

## Checklist
- [X] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [X] My changes are covered by tests

### CODE REVIEW

File: pythreepio/pythreepio/threepio.py [hunk 1]
```diff
@@ -7,25 +7,67 @@
...
 
 class Threepio(object):
+    """
+    Threepio Class to translate a command from one framework to another.
+
+    :param from_lang: Framework to convert from
+    :type from_lang: str
+    :param to_lang: Framework to convert to
+    :type to_lang: str
+    :param framework: Framework to convert to
```
Comment (Nolski) [inline pythreepio/pythreepio/threepio.py]: Something more descriptive might be `Reference to package that represents to_lang`

**Review by Nolski — CHANGES_REQUESTED** [review]
Just one small change and I'll merge :)

File: pythreepio/pythreepio/threepio.py (hunk 1 above)
Comment (Praful932) [reply pythreepio/pythreepio/threepio.py]: Done

**Comment by Praful932** [comment]
@Nolski The [Openmined styleguide](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) mentions Google Doc style while Pysyft uses rst format for docs. To keep it consistent with Pysyft I used rst format, let me know if I need to change the doc style

**Review by ramesht007 — APPROVED** [review]

**Comment by ramesht007** [comment]
Looks good to me :)
Thanks, @Praful932

**Review by Nolski — APPROVED** [review]

**Pull request merged**